src/commands/image-manipulation/petpet.js

```diff
@@ -0,0 +1,25 @@
+const { CanvasTemplates, Command } = require('../../')
+
+const { MessageAttachment } = require('discord.js')
+
+module.exports = class Petpet extends Command {
+  constructor (client) {
+    super({
+      name: 'petpet',
+      aliases: ['patpat'],
+      category: 'images',
+      requirements: { canvasOnly: true },
+      parameters: [{
+        type: 'image',
+        missingError: 'commands:morejpeg.missingImage'
```

Comment:
Create an error string for this command instead of uring the one from morejpeg

Reply:
will do